bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html

```diff
@@ -183,11 +183,11 @@ <h4 class="section-title" id="drumbeat">
       <p>
         {% trans
           logo=static('img/foundation/annualreport/2009/drumbeat-logo.jpg'),
-          webcraft="http://www.drumbeat.org/",
-          festival="http://www.drumbeat.org/",
-          movies="http://www.drumbeat.org/",
-          subtitles="http://www.drumbeat.org/",
-          fund="https://donate.mozilla.org/"
+          webcraft="//www.drumbeat.org/webcraft",
```

Comment:
@alexgibson Has resolved these issues in a separate PR [https://github.com/mozilla/bedrock/pull/3608], the only change here is to use the `donate_url` helper for the `fund` variable